weblate/middleware.py

```diff
@@ -69,10 +69,7 @@
 }
 
 # URLs requiring inline javascript
-INLINE_PATHS = {
-    "social:begin",
-    "djangosaml2idp:saml_login_process",
-}
+INLINE_PATHS = {"social:begin", "djangosaml2idp:saml_login_process", "widgets"}
```

Comment:
Please avoid this, these are third-party apps where we can't avoid that, but our code should not use inline javascript.